// @cloudinary/transformation-builder-sdk/qualifiers/foregroundObject.cjs
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

/**
 * @memberOf Qualifiers
 * @description This namespace contains all the foreground objects used in the SDK
 * @namespace ForegroundObject
 * @example
 * import {ForegroundObject} from '@cloudinary/url-gen/qualifiers/foregroundObject'
 */
var ForegroundObject = {
    AIRPLANE: 'airplane',
    BUS: 'bus',
    DINING_TABLE: 'dining_table',
    SHEEP: 'sheep',
    BICYCLE: 'bicycle',
    CAR: 'car',
    DOG: 'dog',
    SOFA: 'sofa',
    BIRD: 'bird',
    CAT: 'cat',
    HORSE: 'horse',
    TRAIN: 'train',
    BOAT: 'boat',
    CHAIR: 'chair',
    PERSON: 'person',
    TV: 'tv',
    BOTTLE: 'bottle',
    COW: 'cow',
    POTTED_PLANT: 'potted_plant',
    MOTORBIKE: 'motorbike',
};
/**
 * @summary qualifier
 * @description Foreground object "airplane".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var airplane = function () { return ForegroundObject.AIRPLANE; };
/**
 * @summary qualifier
 * @description Foreground object "bus".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var bus = function () { return ForegroundObject.BUS; };
/**
 * @summary qualifier
 * @description Foreground object "dining_table".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var diningTable = function () { return ForegroundObject.DINING_TABLE; };
/**
 * @summary qualifier
 * @description Foreground object "sheep".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var sheep = function () { return ForegroundObject.SHEEP; };
/**
 * @summary qualifier
 * @description Foreground object "bicycle".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var bicycle = function () { return ForegroundObject.BICYCLE; };
/**
 * @summary qualifier
 * @description Foreground object "car".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var car = function () { return ForegroundObject.CAR; };
/**
 * @summary qualifier
 * @description Foreground object "dog".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var dog = function () { return ForegroundObject.DOG; };
/**
 * @summary qualifier
 * @description Foreground object "sofa".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var sofa = function () { return ForegroundObject.SOFA; };
/**
 * @summary qualifier
 * @description Foreground object "bird".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var bird = function () { return ForegroundObject.BIRD; };
/**
 * @summary qualifier
 * @description Foreground object "cat".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var cat = function () { return ForegroundObject.CAT; };
/**
 * @summary qualifier
 * @description Foreground object "horse".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var horse = function () { return ForegroundObject.HORSE; };
/**
 * @summary qualifier
 * @description Foreground object "train".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var train = function () { return ForegroundObject.TRAIN; };
/**
 * @summary qualifier
 * @description Foreground object "boat".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var boat = function () { return ForegroundObject.BOAT; };
/**
 * @summary qualifier
 * @description Foreground object "chair".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var chair = function () { return ForegroundObject.CHAIR; };
/**
 * @summary qualifier
 * @description Foreground object "person".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var person = function () { return ForegroundObject.PERSON; };
/**
 * @summary qualifier
 * @description Foreground object "tv".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var tv = function () { return ForegroundObject.TV; };
/**
 * @summary qualifier
 * @description Foreground object "bottle".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var bottle = function () { return ForegroundObject.BOTTLE; };
/**
 * @summary qualifier
 * @description Foreground object "cow".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var cow = function () { return ForegroundObject.COW; };
/**
 * @summary qualifier
 * @description Foreground object "potted_plant".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var pottedPlant = function () { return ForegroundObject.POTTED_PLANT; };
/**
 * @summary qualifier
 * @description Foreground object "motorbike".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
var motorbike = function () { return ForegroundObject.MOTORBIKE; };

exports.ForegroundObject = ForegroundObject;
exports.airplane = airplane;
exports.bicycle = bicycle;
exports.bird = bird;
exports.boat = boat;
exports.bottle = bottle;
exports.bus = bus;
exports.car = car;
exports.cat = cat;
exports.chair = chair;
exports.cow = cow;
exports.diningTable = diningTable;
exports.dog = dog;
exports.horse = horse;
exports.motorbike = motorbike;
exports.person = person;
exports.pottedPlant = pottedPlant;
exports.sheep = sheep;
exports.sofa = sofa;
exports.train = train;
exports.tv = tv;
