// @cloudinary/transformation-builder-sdk/qualifiers/foregroundObject.js
/**
 * @memberOf Qualifiers
 * @description This namespace contains all the foreground objects used in the SDK
 * @namespace ForegroundObject
 * @example
 * import {ForegroundObject} from '@cloudinary/url-gen/qualifiers/foregroundObject'
 */
const ForegroundObject = {
    AIRPLANE: 'airplane',
    BUS: 'bus',
    DINING_TABLE: 'dining_table',
    SHEEP: 'sheep',
    BICYCLE: 'bicycle',
    CAR: 'car',
    DOG: 'dog',
    SOFA: 'sofa',
    BIRD: 'bird',
    CAT: 'cat',
    HORSE: 'horse',
    TRAIN: 'train',
    BOAT: 'boat',
    CHAIR: 'chair',
    PERSON: 'person',
    TV: 'tv',
    BOTTLE: 'bottle',
    COW: 'cow',
    POTTED_PLANT: 'potted_plant',
    MOTORBIKE: 'motorbike',
};
export { ForegroundObject };
/**
 * @summary qualifier
 * @description Foreground object "airplane".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const airplane = () => ForegroundObject.AIRPLANE;
/**
 * @summary qualifier
 * @description Foreground object "bus".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const bus = () => ForegroundObject.BUS;
/**
 * @summary qualifier
 * @description Foreground object "dining_table".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const diningTable = () => ForegroundObject.DINING_TABLE;
/**
 * @summary qualifier
 * @description Foreground object "sheep".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const sheep = () => ForegroundObject.SHEEP;
/**
 * @summary qualifier
 * @description Foreground object "bicycle".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const bicycle = () => ForegroundObject.BICYCLE;
/**
 * @summary qualifier
 * @description Foreground object "car".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const car = () => ForegroundObject.CAR;
/**
 * @summary qualifier
 * @description Foreground object "dog".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const dog = () => ForegroundObject.DOG;
/**
 * @summary qualifier
 * @description Foreground object "sofa".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const sofa = () => ForegroundObject.SOFA;
/**
 * @summary qualifier
 * @description Foreground object "bird".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const bird = () => ForegroundObject.BIRD;
/**
 * @summary qualifier
 * @description Foreground object "cat".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const cat = () => ForegroundObject.CAT;
/**
 * @summary qualifier
 * @description Foreground object "horse".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const horse = () => ForegroundObject.HORSE;
/**
 * @summary qualifier
 * @description Foreground object "train".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const train = () => ForegroundObject.TRAIN;
/**
 * @summary qualifier
 * @description Foreground object "boat".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const boat = () => ForegroundObject.BOAT;
/**
 * @summary qualifier
 * @description Foreground object "chair".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const chair = () => ForegroundObject.CHAIR;
/**
 * @summary qualifier
 * @description Foreground object "person".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const person = () => ForegroundObject.PERSON;
/**
 * @summary qualifier
 * @description Foreground object "tv".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const tv = () => ForegroundObject.TV;
/**
 * @summary qualifier
 * @description Foreground object "bottle".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const bottle = () => ForegroundObject.BOTTLE;
/**
 * @summary qualifier
 * @description Foreground object "cow".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const cow = () => ForegroundObject.COW;
/**
 * @summary qualifier
 * @description Foreground object "potted_plant".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const pottedPlant = () => ForegroundObject.POTTED_PLANT;
/**
 * @summary qualifier
 * @description Foreground object "motorbike".
 * @memberOf Qualifiers.ForegroundObject
 * @return {Qualifiers.ForegroundObject}
 */
export const motorbike = () => ForegroundObject.MOTORBIKE;
